Packs/Sysdig/Integrations/Sysdig/Sysdig.py

```diff
@@ -0,0 +1,353 @@
+from datetime import datetime
+
+import demistomock as demisto  # noqa: F401
+from CommonServerPython import *  # noqa: F401
+from sdcclient import SdMonitorClient, SdSecureClient
+
+
+class Client(BaseClient):
+    def query_sysdig(self, url_suffix):
+        return self._http_request(
+            method='GET',
+            url_suffix=url_suffix
+        )
+
+# Convert from Unix time to ISO
+
+
+def time_convert(dtString):
+    try:
+        dtConverted = datetime.fromtimestamp(int(str(dtString).split('.')[0])).isoformat()
+    except Exception:
+        dtConverted = datetime.fromtimestamp(int(str(int(dtString / 1000)).split('.')[0])).isoformat()
+    return dtConverted
+
+# Flatten json structures
+
+
+def flatten_data(y):
+    out = {}
+
+    def flatten(x, name=''):
+        if type(x) is dict:
+            for a in x:
+                flatten(x[a], name + a + '_')
+        elif type(x) is list:
+            i = 0
+            for a in x:
+                flatten(a, name + str(i) + '_')
+                i += 1
+        else:
+            out[name[:-1]] = x
+    flatten(y)
+    return out
+
+
+def get_alerts(monitor, url):
+    sdclient = SdMonitorClient(token=monitor, sdc_url=url)
+    ok, res = sdclient.get_alerts()
+    if not ok:
+        return res
+    results = []
+    for alert in res['alerts']:
+        results.append(alert)
+    readable_output = tableToMarkdown('Sysdig Alerts', results, removeNull=True)
+    return CommandResults(
+        readable_output=readable_output,
+        outputs_prefix='Sysdig.Alerts',
+        outputs_key_field='Alerts',
```

Comment:
how the alerts json looks like, since `outputs_key_field` should be the property name which makes the alert unique.
For example if the json looks like:
```
[{ "alert_id": 100, "name": "some name"}]
```

then `outputs_key_field='alert_id',`